lib/poison/encoder.ex

```diff
@@ -118,6 +118,14 @@ defimpl Poison.Encoder, for: BitString do
     [seq(char) | escape(rest, :javascript)]
   end
 
+  defp escape(<<char :: utf8>> <> rest, :html_safe) when char in '/' do
```

Comment:
You can match directly on the `/` in the pattern instead of using a guard: `<<?/ :: utf8>>`.